app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(comment_params)
```

Comment:
Great job here with your Comments Controller! Nice job keeping it "skinny" and only defining methods(the create method) that you are using in your app